pkgs/by-name/lo/longcat/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
let
  version = "0.0.12";
in
buildGoModule {
```

`rec` expression is a anti-pattern and should be avoid, refers to [best practices](https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec) (https://github.com/NixOS/nixpkgs/issues/315337)